src/forms/__stories_/FormErrorMessage.stories.mdx

```diff
@@ -0,0 +1,76 @@
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs"
```

Comment:
I love the look and feel of these modern storybook pages. One thing I think we might run into though is with the React code embedded into the mdx files we won't know if there is an error. For example if we change the component in a way that requires a change in its consumption we could easily miss updating it in the story. The type checking is nice in .jsx files. Curious on your thoughts

Reply:
@ludtkemorgan Good points…there might still be a way to add type checking for MDX files, even if it's just a custom linting step. I can do some more research.

Reply:
After the recent round of evaluation I think we (@emilyjablonski and I) are leaning towards going back to `.tsx` for stories rather than including them in MDX files, so probably once we finish all the design system docs eval we can address this.